flowbased-computation/flowbased-computation-api/src/main/java/com/farao_community/farao/flowbased_computation/tools/FlowbasedComputationTool.java

```diff
@@ -155,14 +157,15 @@ public void run(CommandLine line, ToolRunningContext context) throws IOException
         UcteAliasesCreation.createAliases(network);
         RaoInputHelper.cleanCrac(crac, network);
         crac.synchronize(network);
-        //TODO : handling also Ucte format
-        GlskProvider cimGlskProvider = new CimGlskProvider(new FileInputStream(glskFile.toFile()), network, instant);
+        // TODO: problem here: Cim or Ucte ??
+        GlskDocumentLinearGlskConverter converter = new UcteGlskDocumentLinearGlskConverter();
+        GlskProvider glskProvider = new ChronologyGlskProvider(converter.convert(new FileInputStream(glskFile.toFile()), network), instant);
```

Comment:
@murgeyseb I don't know how to solve this! I changed UcteGlskDocumentLinearGlskConverter and CimGlskDocumentLinearGlskConverter so that they implement GlskDocumentLinearGlskConverter, but I had to make them not be utility classes anymore. And now, I'm stuck :cry: